miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -348,14 +349,14 @@ private static DetailedSample toDetailedSample(DetailedSampleDto from) {
   }
 
   /**
-   * Extracts parent details from the DTO, according to three possible cases:
+   * Extracts parent details from the DTO, according to these possible cases:
    * 
    * <ol>
    * <li>parent ID is provided. This implies that the parent exists, so no other parent information will be required</li>
-   * <li>identity information and parentSampleClassId are provided. This implies that a tissue parent should be created, and that the
+   * <li>identity information and parentTissueSampleClassId are provided. This implies that a tissue parent should be created, and that the
    * identity may or may not yet exist</li>
-   * <li>identity information is provided, but no parentSampleClassId. This implies that the direct parent of this sample will be an
-   * identity, which may or may not yet exist</li>
+   * <li>identity information is provided, but no parentTissueSampleClassId. This implies that the direct parent of this sample will be an
+   * identity, which may or may not yet exist. If the sampleClassId is an aliquot, a stockClassId must be provided.</li>
```

Comment:
But if the sampleClassId is an aliquot, then a missing parentTissueSampleClassId would be invalid... right? I think the "If the sampleClassId is an aliquot, a stockClassId must be provided." bit belongs with the previous point ("identity information and parentTissueSampleClassId are provided.")